### PR TITLE
Added comments

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
@jainaman224 I was not able to delete those previous commits from my last PR #690. So, I made this new PR. I tried to rebase those commits from my local repo and I successfully did also. But was not able to reflect those changes in the PR. Kindly review and merge this one and provide me advice over the same.

### CODE REVIEW

File: Burrows_Wheeler_Transformation_Algorithm/BWT.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    assert "\002" not in s and "\003" not in s, "Input string cannot contain STX and ETX characters"
+    s = "\002" + s + "\003"
+    table = sorted(s[i:] + s[:i] for i in range(len(s)))
+    last_column = [row[-1:] for row in table]
+    return "".join(last_column)
+
+
+#----INVERSE BURROWS WHEELER TRANSFORMATION
+#This is to get back the original message out of the compressed version.
+def ibwt(r):
```
Comment (jainaman224) [inline Burrows_Wheeler_Transformation_Algorithm/BWT.py]: What is `ibwt`?

**Comment by jainaman224** [comment]
Please close the other PR.

File: Burrows_Wheeler_Transformation_Algorithm/BWT.py (hunk 1 above)
Comment (Priyankasaggu11929) [reply Burrows_Wheeler_Transformation_Algorithm/BWT.py]: @jainaman224, ibwt or Inverse Burrows-Wheeler Transformation is to compute the inverse of BWT i.e. to undo the effects of BWT and recover the original string.

**Comment by Priyankasaggu11929** [comment]
> Please close the other PR.

File: Burrows_Wheeler_Transformation_Algorithm/BWT.py (hunk 1 above)
Comment (Priyankasaggu11929) [reply Burrows_Wheeler_Transformation_Algorithm/BWT.py]: And I closed the previous PR #690 also.

**Pull request merged**